src/agrag/modules/embedding/README.md

```diff
@@ -6,28 +6,43 @@ Here are the configurable parameters for this module:
 
 ```
 embedding:
-  embedding_model: The name of the Huggingface or Bedrock model to use for generating embeddings (default is "BAAI/bge-large-en" from Huggingface). Currently only Amazon Titan and Cohere Embedding models are supported on Bedrock.
+  embedding_model: The name of theEmbedding model to use for generating embeddings (default is "BAAI/bge-large-en" from Huggingface). 
+
+  embedding_model_platform: The name of the platform where the model is hosted. Currently only Huggingface ("huggingface") and Bedrock ("bedrock") models are supported. 
+  Currently only Cohere (https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-embed.html) and Amazon Titan (https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-titan.html) embedding models are supported on Bedrock.
+  
+  embedding_model_platform_args: Additional platform-specific parameters to use when initializing the model, generating embeddings, etc.
   
   pooling_strategy: The strategy to use for pooling embeddings. Options are 'mean', 'max', 'cls' (default is None).
   
   normalize_embeddings: Whether to normalize the embeddings.
   
-  hf_model_params: Additional parameters to pass to the Huggingface model's `from_pretrained` initializer method.
```

Comment:
there should have a document to explain those parameters, not just as comments.

Reply:
Sure, I will add this to the `docs` PR: https://github.com/autogluon/autogluon-rag/pull/41

Let's merge this in first?